rs/state_machine_tests/src/lib.rs

```diff
@@ -847,6 +847,7 @@ pub struct StateMachine {
     /// A drop guard to gracefully cancel the ingress watcher task.
     _ingress_watcher_drop_guard: tokio_util::sync::DropGuard,
     query_stats_payload_builder: Arc<PocketQueryStatsPayloadBuilderImpl>,
+    time_of_last_round: RwLock<SystemTime>,
```

Comment:
Could you explain to me the relationship of the fields `time` and `time_of_last_round` and the two getters that use them, namely `time()` and `time_of_next_round()`?

Reply:
The `time` is the time that will determine the time of the next round and users can advance it. The `time_of_last_round` is the time of the last executed round.

Reply:
Ah ok I think I get it now. Could you maybe move `time_of_last_round` next to `time` and add a comment?

Reply:
Done.